src/js/view/view.js

```diff
@@ -602,9 +577,7 @@ define([
             _controlsLayer.appendChild(_controlbar.element());
 
             _playerElement.onfocusin = handleFocus;
-            _playerElement.onfocusout = handleBlur;
             _playerElement.addEventListener('focus', handleFocus);
```

Comment:
Let's only listen to focusin for showing the controlbar:

` _playerElement.addEventListener('focusin', handleFocus);`